velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -245,6 +249,17 @@ TEST_F(StringTest, Ascii) {
   EXPECT_EQ(ascii(std::nullopt), std::nullopt);
 }
 
+TEST_F(StringTest, BitLength) {
```

Comment:
Not sure whether the community prefers lower case for the initial letter of test name.  I note many tests in this file use upper case. @mbasmanova

Reply:
Test names should start with lower case. It would be nice to create a separate PR to fix test names in this file.

Reply:
I can raise a PR to do this.

Reply:
Changed this test to start with lower case.

Reply:
Created a PR here - https://github.com/facebookincubator/velox/pull/9028